AppCenterLoaderApp/appcenter-loader/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.microsoft.appcenter.loader">
-    <application android:allowBackup="true" android:label="@string/app_name"
-        android:supportsRtl="true">
+    <application android:label="@string/app_name">
```

Comment:
Hi! Thanks for your contribution PR. Please update the changelog with mention what exactly these changes fix.

Reply:
Thanks, I've updated the changelog in 955dc9c.